### DISCUSSION TITLE
Supabase Row level Security AI not work

### DISCUSSION THREAD

**Discussion by Othman-Sami** [supabase/supabase#21500] [Questions]
Hi 

I use Supabase old RLS very Well but now its show new AI tools for make 

but nothing work when I told AI and use the answer 

I have table called "user"
I want to all access if the user authenticated and just current user = "test@test.com"
but nothing work 

this is the AI code :

CREATE POLICY "control_user"
ON "user"
FOR ALL TO auth.uid ()
WITH
  CHECK ("test@test.com" = auth.uid ());

**Comment by GaryAustin1** [comment]
Toggle the feature preview back off to get the "old" methods.  Sort of surprised it does not give a choice.
